index.js

```diff
@@ -0,0 +1,298 @@
+'use strict'
+
+const _ = require('lodash')
+const childProcess = require('child_process')
+const fs = require('fs-extra')
+const path = require('path')
+
+const pkg = require('./package.json')
+const logger = require('debug')(pkg.name)
+
+const promisify = require('es6-promisify')
+const writeFile = promisify(fs.writeFile)
+const mkdirs = promisify(fs.mkdirs)
+const copy = promisify(fs.copy)
+const symlink = promisify(fs.symlink)
+const tmpdir = promisify(require('tmp').dir)
+
+function kebabify (object) {
```

Comment:
I'm pretty sure by naming functions like `kebabify` I can ingratiate myself into the node community.